mobile-app/app/screens/AppNavigator/screens/Dex/components/DexScrollable.tsx

```diff
@@ -156,6 +156,7 @@ function TotalLiquidityValue({
           decimalScale={0}
```

Comment:
let's not use `decimalScale`, instead we control the decimal places using `BigNumber`.
At line 155, use this to display currency `getPrecisedCurrencyValue(value)`

Reply:
updated line 155 to :
`value={BigNumber(value).toFixed(2)}`
removed `decimalScale={0}`